From 8aed8e9f8c255dca05258b93cb8456f0703d2ad9 Mon Sep 17 00:00:00 2001
From: Huy Hoang <boydaihungst@gmail.com>
Date: Sun, 15 Oct 2023 06:20:10 +0700
Subject: [PATCH 3/3] disable back panel touch area

---
 rw-system.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/rw-system.sh b/rw-system.sh
index 3302715..d093e69 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -732,6 +732,11 @@ if getprop ro.vendor.build.fingerprint | grep -q -e nubia/NX669; then
     sku="$(getprop ro.boot.product.vendor.sku)"
     mount /vendor/etc/audio/sku_${sku}_qssi/audio_policy_configuration.xml /vendor/etc/audio/sku_$sku/audio_policy_configuration.xml
     chmod 0666 /sys/kernel/lcd_enhance/hbm_state
+    # Disable back panel touch (which would hide keyboard if accidentally touched)
+    deviceName=$(cat /sys/class/input/event9/device/name)
+    if [[ $deviceName == "nubia_sar0_channel0" ]]; then
+      rm -rf /dev/input/event9
+    fi
 fi
 
 # For ZF8, the "best" audio policy isn't the one for QSSI
-- 
2.42.0

